bin/ncar_build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Assume that this script is run from the root directory
+
+# If an argument is supplied and if it's 'GPU' (case insensitive), assign that to the variable 
+#    in CMakeLists.txt file under the root directory. Default is CPU mode.
+MODE=${1^^}
+if [ "$MODE" == "GPU" ]; then
+  sed -i 's/MODE CPU/MODE GPU/g' CMakeLists.txt
+  # Somehow the Release mode will break the GPU code; change back to Debug mode
+  sed -i 's/CMAKE_BUILD_TYPE Release/CMAKE_BUILD_TYPE Debug/g' CMakeLists.txt
+  if [ "$NCAR_HOST" == "casper" ]; then
+    # Use cc70 for V100 GPU on Casper
+    sed -i 's/cc80/cc70/g' CMakeLists.txt
+  fi
+else
+  sed -i 's/MODE GPU/MODE CPU/g' CMakeLists.txt     # This will switch it to CPU if it was GPU
+fi
+
+# If we are going to measure the performance for Supercell and Hurricane,
+# we only want to run the first 20 iterations for each case.
```

Comment:
@sjsprecious the paper has results for both 20 iterations and the full number.  It is not a good idea to force the change to 20 iterations here. If we only look at 20 iterations, we will likely miss numerical problems that cause the code to fail to converge in the expected number of iterations.

Reply:
Good point. I have removed this hardcode change.